packages/components/src/components/telekom/telekom-footer/telekom-footer.css

```diff
@@ -76,7 +75,7 @@
 
 .scale-telekom-footer :where(a:focus-visible) {
   outline: var(--telekom-line-weight-highlight) solid
-    var(--telekom-color-functional-focus);
+    var(--telekom-color-functional-focus-standard-on-dark);
```

Comment:
Don't forget to also rename to `--telekom-color-functional-focus-standard-on-dark-background` 🙏